temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -40,20 +40,18 @@ message CodecServerSpec {
     bool include_cross_origin_credentials = 3;
 }
 
-message GlobalNamespaceSpec {
-    // The regions where the namespace is (or will be) located.
-    // Optional, default is empty.
-    repeated string passive_regions = 1;
-}
-
 message NamespaceSpec {
     // The name to use for the namespace.
     // This will create a namespace that's available at '<name>.<account>.tmprl.cloud:7233'.
     // The name is immutable. Once set, it cannot be changed.
     string name = 1;
-    // The region where the namespace is (or will be) active.
-    // The region is immutable. Once set, it cannot be changed.
-    string region = 2;
+    // The regions of availability for the namespace.
+    // Specifying more than one region makes the namespace "global", which is currently a preview only feature with restricted access.
+    // Please reach out to Temporal support for more information on global namespaces.
+    // When provisioned the global namespace will be active on the first region in the list and passive on the rest.
+    // Number of supported regions is 2. 
+    // The regions is immutable. Once set, it cannot be changed.
+    repeated string regions = 2;
```

Comment:
do you want to use the RegionID which contains cloud provider and region name?

Reply:
added a region id.